app/routes/api/stage-execution.js

```diff
@@ -51,6 +51,7 @@ module.exports = [{
         }).allow(null).optional()
       }),
       failAction: async (_request, h, err) => {
+        console.log('Error when validating payload: ', err)
```

Comment:
bit better would be:

console.log(`Error when validating payload: ${JSON.stringify({
          errorMessage: error.message,
          payload: request.payload
        })}`)

sample outcome:

Error when validating payload: {"errorMessage":"\"confirm\" does not contain 1 required value(s)","payload":{"reference":"AHWR-555A-FD4C","confirm":["sentChecklist"],"page":1}}